conda_build_all/builder.py

```diff
@@ -210,6 +210,13 @@ def compute_build_distros(self, index, recipes):
         return all_distros
 
     def main(self):
+        # We set the CONDA_NPY and CONDA_PY variables to meaningless values.
+        # Since we compute a build matrix anyway, these are only useful to
+        # prevent conda-build throwing a wobbly for them missing.
+        import conda_build.config
+        conda_build.config.config.CONDA_NPY = 10
+        conda_build.config.config.CONDA_PY = 10
```

Comment:
So I understand the need to set the NumPy version, but I thought the Python version already defaulted to the version of Python in `root`. Is there some case where that is not true?